exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -235,6 +235,37 @@ open class Table(name: String = ""): ColumnSet(), DdlAware {
         return newColumn
     }
 
+    var isCustomPKNameDefined = false
```

Comment:
Consider removing

Reply:
Hi @Tapac , 
As you know, If we have one single column in PK, Exposed adds "PRIMARY KEY" keyword in front of the column without defining a constraint.
I use isCustomPKNameDefined variable to check if a custom pk name is defined.  So if it's set to true, I force defining a primary key CONSTRAINT with the specified name.
Do you think that this not what we need or I better find an other way without using a dedicated variable(isCustomPKNameDefined)? Thanks !